bpf/headers/maps.h

```diff
@@ -18,7 +18,7 @@ limitations under the License.
 
 struct bpf_map __section("maps") cookie_original_dst = {
     .type = BPF_MAP_TYPE_LRU_HASH,
-    .key_size = sizeof(__u32),
+    .key_size = sizeof(__u64),
```

Comment:
https://github.com/merbridge/merbridge/blob/6db5f4d1b2c3b6af440683763c55b6713d33d1c0/bpf/Makefile#L70

should be also updated.

Reply:
Thank you! Fixed